README.md

```diff
@@ -1,2 +1,74 @@
 # Awesome-Books
-This an app created using javascript
+
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+> This project is a book collection and shuffling application.
+
+## Description
+
+## Hosting
+
+This Portfolio is hosted on GitHub Page. [Click here](https://summyalena.github.io/Awesome-Books/)
+
+### Mobile version Screenshot:
+
+
+
+### Desktop version Screenshot:
+
+
+
+### Objective
+
+This project's goals are as follows:
+
+## Built With
+
+- HTML, CSS, Javascript (Vanilla)
+
+## Getting Started
+  This is a single-page application that lets one add his/her favorite books and deletes if needed. Do well to give a star 🌟 as we progress on a journey to becoming better.
+
+### Prerequisites
+
+Basic prequisites for opening a webpage as is required.
+
+- A standard PC -
+- Compatible and up to data browser both for PC and mobile.
+- Installed Code Editor
+- 
+### Usage
+
+This code can be used for educational 📘 and code viewing purposed. The copying of these code for other use would be liable to plagarism and practically and developmentally unaccceptable.
+
+## Authors
+
+👤 **Alena Okolike.**
+
+- GitHub: [@summyalena](https://github.com/summyalena)
+- Twitter: [@AOkolike](https://twitter.com/AOkolike)
+
+**Tinaye Tinaye**
+-Twitter: 
+## 🤝 Contributor
+**None**
+
+Contributions, issues, and feature requests are welcome!
+
+If you find any bug or suggestion for improvement please Feel free to check the [issues page](../../issues/) with your proposed change.
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Hat tip to ![](https://img.shields.io/badge/Microverse-blueviolet) is where you Gain skills you need to get hired. Learn Online and connect to life changing opportunities. [Apply today](https://www.microverse.org/?grsf=uv064g)
+Design acknowledgment goes to ###Cindy Shin in Behance [Click here to see design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015)
+## Update info
+
+
+## 📝 License
+
+This project is [MIT](./MIT.md) licensed.
+
```

Comment:
- [ ] I noticed you added `LICENSE` file to your project Great job 💯 but the link to `MIT.md` file is not working, I think this happened because you have to rename the `LICENSE` file to the `MIT.md` file, Please rename it Make sure this link works 👌

> The MIT license gives users express permission to reuse code for any purpose, sometimes even if the code is part of proprietary software. As long as users include the original copy of the MIT license in their distribution, they can make any changes or modifications to the code to suit their own needs. 💯

Reply:
## Resolved 😅 

- updated the markdown to link to the `LICENSE` file
- now the link is working fine
- thanks for noticing
- resolved also some additional README.md errors

![Thor Shoulders](https://media.giphy.com/media/EfsqyinraWHCKWxsjq/giphy.gif)